cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -127,22 +130,37 @@ private boolean setRegulatingControl(CgmesRegulatingControlForStaticVarCompensat
         }
         svc.setRegulatingTerminal(regulatingTerminal);
 
-        svc.setProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "RegulatingControl", rc.regulatingControlId);
         return okSet;
     }
 
-    private void setDefaultRegulatingControl(CgmesRegulatingControlForStaticVarCompensator rc, StaticVarCompensator svc, boolean onlyReactivePowerReg) {
+    private boolean isValidVoltageFromRegulatingControl(RegulatingControl control) {
+        return control != null
+                && RegulatingControlMapping.isControlModeVoltage(control.mode)
+                && Double.isFinite(control.targetValue)
+                && control.targetValue > 0;
+    }
+
+    private boolean isValidReactivePowerFromRegulatingControl(RegulatingControl control) {
+        return control != null
+                && RegulatingControlMapping.isControlModeReactivePower(control.mode)
+                && Double.isFinite(control.targetValue);
+    }
+
+    private void setDefaultRegulatingControl(CgmesRegulatingControlForStaticVarCompensator rc, StaticVarCompensator svc, boolean onlyReactivePowerReg, RegulatingControl control) {
 
         double targetVoltage = Double.NaN;
         double targetReactivePower = Double.NaN;
         StaticVarCompensator.RegulationMode regulationMode;
 
+        // Before using the default target value,
+        // We try to keep data from original regulating control if available.
+        // Even if the regulating control was disabled it may have defined a valid target value
         if (RegulatingControlMapping.isControlModeVoltage(rc.defaultRegulationMode.toLowerCase())) {
             regulationMode = onlyReactivePowerReg ? StaticVarCompensator.RegulationMode.OFF : StaticVarCompensator.RegulationMode.VOLTAGE;
-            targetVoltage = rc.defaultTargetVoltage;
+            targetVoltage = isValidVoltageFromRegulatingControl(control) ? control.targetValue : rc.defaultTargetVoltage;
```

Comment:
When you are here, you already know that it is a VOLTAGE control mode (on or off).

Reply:
when defining the default regulating control we may be establishing a different kind of regulation that was defined in the original control. This is why we check if the mode defined in the RC is consistent with the one being defined now. I ran into an inconsistency with one of the unit tests we had, this is why this condition has been added.